nemo_skills/training/nemo_rl/start_grpo.py

```diff
@@ -70,54 +70,35 @@ def parse_args() -> tuple[argparse.Namespace, list[str]]:
 
 from nemo_rl.data.interfaces import TaskDataSpec
 
+def extract_dataset(split, output_key, dataset_path):
+    if not dataset_path.startswith('/'):
+        original_ds = load_dataset(dataset_path, split=split)
+    else:
+        import pandas as pd
+        from datasets import Dataset
+        df = pd.read_json(dataset_path, lines=True, dtype={output_key: str})
+        df = df[['problem', output_key]]
+        original_ds = Dataset.from_pandas(df)
+    return original_ds
 
 def format_math(data, output_key: str = "expected_answer"):
     return {
-        "messages": [
-            {
-                "role": "user",
-                "content": data["problem"],
-            },
-            {
-                "role": "assistant",
-                "content": data[output_key],
-            },
-        ],
+        "problem": data["problem"],
```

Comment:
can we make this independent of the particular key (problem here). So we would just pass in all of the data as input to prompt.fill and whatever it returns we use as an input. And for the output, we'd ask user to provide a key (can be expected_answer) as default. That way it would work with any prompt we have and not assume a particular structure, right?